src/pages/search/registration_search/RegistrationSearchBar.tsx

```diff
@@ -368,7 +366,7 @@ interface SelectedContributorFacetButtonProps {
   personIdentifier: string;
 }
 
-const SelectedContributorFacetButton = ({ personIdentifier }: SelectedContributorFacetButtonProps) => {
+export const SelectedPersonFacetButton = ({ personIdentifier }: SelectedContributorFacetButtonProps) => {
```

Comment:
Samme som nevnt i tidligere PR, så burde nok disse flyttes til egne filer på et tidspunkt, men tenker det er like greit å ta det når vi endrer fasetthåndtering for resultat, som kommer i senere PR